github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -130,6 +130,32 @@ presubmits:
           requests:
             memory: "8Gi"
 
+  - name: check-provision-k8s-1.13.3
+    always_run: false
+    optional: true
+    decorate: true
+    max_concurrency: 1
+    spec:
```

Comment:
@rmohr Say do we miss indentation? I can see double `spec` on the same indentation level.

Reply:
Yes you are right. The `spec:` line itself should be removed.